apps/vx-scan/frontend/src/app_tally_report_paths.test.tsx

```diff
@@ -348,16 +341,15 @@ test('printing: polls closed, primary election, all precincts + quickresults on'
     electionMinimalExhaustiveSampleWithReportingUrlDefinition;
   const { election } = electionDefinition;
   apiMock.expectGetConfig({ electionDefinition, pollsState: 'polls_open' });
-  apiMock.expectGetScannerStatus({ ...statusNoPaper, ballotsCounted: 3 }, 3);
-  const { card } = renderApp({ connectPrinter: true });
+  apiMock.expectGetScannerStatus({ ...statusNoPaper, ballotsCounted: 3 }, 4);
```

Comment:
One notable pain point while updating these tests was having to tweak get scanner status counts, through a process of guess and check. Why this was necessary I'm still not exactly sure but so many things can affect test duration and by extension these counts.

This, in conjunction with my manual mocking of auth status, is making think that a built-in solution for mocking polled methods is worthwhile